README.md

```diff
@@ -22,27 +22,27 @@ The app defines following CRUD APIs.
 | GET    | /suppliers/{supplierId}/profile              | Get a supplier profile by Id     | |
 | GET    | /suppliers/{supplierId}/products             | Get all products from a supplier | |
 | GET    | /suppliers/{supplierId}/products/{productId} | Get a product by from a supplier | |
-| POST	 | /suppliers/{supplierId}/products             |	Create a product by supplierId   | |
+| POST   | /suppliers/{supplierId}/products             |    Create a product by supplierId   | |
 | POST   | /suppliers                                   | Create a supplier                | |
 | PUT    | /suppliers/{supplierId}                      | Edit a supplier by supplierId    | |
 | DELETE | /suppliers/{supplierId}                      | Delete a supplier by supplierId  | |
 
 ### Categories
 
-| Method | Url                      | Description                                    | Sample Valid Request Body |
-|--------|--------------------------|------------------------------------------------| ------------------------- |
-| GET    | /categories/{categoryId} | Get a category and subcategories by categoryId | |
-| GET    | /categories              | Get all categories and subcategories           | |
-| POST   | /categories/{categoryId} | Create a category by category Id               | |
-| PUT    | /categories/{categoryId} | Edit a category by category Id                 | |
-| DELETE | /categories/{categoryId} | Delete a category by category Id               | |
+| Method | Url                                    | Description                                    | Sample Valid Request Body |
+|--------|----------------------------------------|------------------------------------------------| ------------------------- |
+| GET    | /categories/{categoryId}               | Get a category and subcategories by categoryId | |
+| GET    | /categories                            | Get all categories and subcategories           | |
+| POST   | /categories/{categoryId}               | Create a category by category Id               | |
+| POST   | /categories/{categoryId}/subCategories | Create a subcategory for a given category      | |
```

Comment:
@Geofery 

`| POST   | /subCategories/{categoryId} | Create a subCategories by subCategories Id | |`

bliver til:

`| POST   | /categories/{categoryId}/subCategories | Create a subcategory for a given category      | |`